### ISSUE TITLE
[UX] Stuck on "Render Local Test" without any error message.

### ISSUE THREAD

**Issue by golemUser1** [golemfactory/clay#1896]
I was trying out to submit another Test task, but that was a patience Test...
I am running a Windows system with Nvidia graphics card, which is the Worst case for Blender Tasks, because no GPU Acceleration is momentanly possible.
The "Render Local Test" steps took **several minutes**,  before i was able to submit the task. (And it must be done before submitting a task)
During this time, the gui looks like crahsed! 
can you add a progress meter, or at least a progress animation for the "Render Local Test".(You have one for benchmark computation.)
By the way, the "Render Local Test" UI Element looks like a button, which is confsing, because nothing happens, when i click it.

I add the demo task folder here, if anyone else with a windows system want to test.
[Golem-Airplane.zip](https://github.com/golemfactory/golem/files/1627878/Golem-Airplane.zip)

**Comment by golemUser1** [comment]
it seems that i found the issue behind this myself:
![grafik](https://user-images.githubusercontent.com/35274214/35248847-f410d8dc-ffcf-11e7-9aef-efcce2320c8c.png)
one of the virtual machines had crashed. possibly because of too less free ram. Therefore the task is stuck in the "render local test" stage untill the machine is restarted. 

there is feedback missing for this state!

**Comment by badb** [comment]
The crashed default virtual machine should not be an issue - Golem doesn't use default virtual machine. 
@golemUser1 Can you please write us how much RAM have you assign to golem node and how many cpu cores?

**Comment by golemUser1** [comment]
ok. thats a strange coincidende then. 
i guess this default machine was a relict from the docker quickstart terminal.

I wanted to assign:
4/8 CPU cores (including virtual) 3/8 GB Ram 400GB/1128 GB Disk 
which can be reached when the computer is totally idle.

I tried to reproduce the issue and did some stresstesting.
today it only appeared once in a lot of testing. 

its something sporadic, i will investigate further, maybe it was caused by something else.
